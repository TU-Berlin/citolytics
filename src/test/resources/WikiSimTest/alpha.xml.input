<page>
    <title>Article A</title>
    <ns>0</ns>
    <id>358</id>
    <revision>
        <id>625906990</id>
        <parentid>625773078</parentid>
        <timestamp>2014-09-17T05:10:48Z</timestamp>
        <contributor>
            <ip>114.77.20.119</ip>
        </contributor>
        <comment>/* Religion */</comment>
        <text xml:space="preserve">
            [[Article B]]

            TodayÂ [[SeeAlso Article 1]] Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French. [[Article C]]

            Today Algeria contains, [[SeeAlso Article 2]] in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French.

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in [[Article E]] Arabic and French.
            [[Article D]]

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French.

        </text>
    </revision>
</page>

<page>
<title>Article B</title>
<ns>0</ns>
<id>358</id>
<revision>
    <id>625906990</id>
    <parentid>625773078</parentid>
    <timestamp>2014-09-17T05:10:48Z</timestamp>
    <contributor>
        <ip>114.77.20.119</ip>
    </contributor>
    <comment>/* Religion */</comment>
    <text xml:space="preserve">


            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French. [[Article C]]

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, [[SeeAlso Article 1]]but also the universal literary heritage in Arabic and French.

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in [[Article E]] Arabic and French.

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French.

        </text>
</revision>
</page>

<page>
<title>Article C</title>
<ns>0</ns>
<id>358</id>
<revision>
    <id>625906990</id>
    <parentid>625773078</parentid>
    <timestamp>2014-09-17T05:10:48Z</timestamp>
    <contributor>
        <ip>114.77.20.119</ip>
    </contributor>
    <comment>/* Religion */</comment>
    <text xml:space="preserve">
            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French. [[SeeAlso Article 1]]

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French.

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in [[Article A]] Arabic and French.
            [[Article D]]

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French.

        </text>
</revision>
</page>

<page>
<title>Article D</title>
<ns>0</ns>
<id>358</id>
<revision>
    <id>625906990</id>
    <parentid>625773078</parentid>
    <timestamp>2014-09-17T05:10:48Z</timestamp>
    <contributor>
        <ip>114.77.20.119</ip>
    </contributor>
    <comment>/* Religion */</comment>
    <text xml:space="preserve">
            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French.

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French.

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in [[Article E]] Arabic and French.
            [[Article B]]

            Today Algeria contains, in its literary landscape, big names [[Article a]] having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French.

        </text>
</revision>
</page>

<page>
<title>Article E</title>
<ns>0</ns>
<id>358</id>
<revision>
    <id>625906990</id>
    <parentid>625773078</parentid>
    <timestamp>2014-09-17T05:10:48Z</timestamp>
    <contributor>
        <ip>114.77.20.119</ip>
    </contributor>
    <comment>/* Religion */</comment>
    <text xml:space="preserve">


            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French. [[Article C]]

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French.

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French.
            [[Article A]]

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French. [[Article b]]

        </text>
</revision>
</page>

<page>
<title>SeeAlso Article 1</title>
<ns>0</ns>
<id>358</id>
<revision>
    <id>625906990</id>
    <parentid>625773078</parentid>
    <timestamp>2014-09-17T05:10:48Z</timestamp>
    <contributor>
        <ip>114.77.20.119</ip>
    </contributor>
    <comment>/* Religion */</comment>
    <text xml:space="preserve">

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature,[[SeeAlso Article 2]] but also the universal literary heritage in Arabic and French. [[Article C]]

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French.

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage [[Article A]] in [[Article B]] Arabic and French.


            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French.

            ==See also==
            {{Contains Tifinagh text|compact=yes}}
            * [[Article A]]
            * [[Article B]]

            ==Notes and references==
        </text>
</revision>
</page>

<page>
<title>SeeAlso Article 2</title>
<ns>0</ns>
<id>358</id>
<revision>
    <id>625906990</id>
    <parentid>625773078</parentid>
    <timestamp>2014-09-17T05:10:48Z</timestamp>
    <contributor>
        <ip>114.77.20.119</ip>
    </contributor>
    <comment>/* Religion */</comment>
    <text xml:space="preserve">

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French.

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French.

            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage [[Article E]] in [[Article B]] Arabic and French.


            Today Algeria contains, in its literary landscape, big names having not only marked the Algerian
            literature, but also the universal literary heritage in Arabic and French.

            ==See also==

            * [[Article D]]
            * [[Article b]]

            ==Notes and references==
        </text>
</revision>
</page>


<page>
<title>Article a</title>
<ns>0</ns>
<id>11858</id>
<redirect title="Article A"/>
<revision>
    <id>518833992</id>
    <parentid>511689418</parentid>
    <timestamp>2012-10-20T06:12:35Z</timestamp>
    <contributor>
        <username>M0rphzone</username>
        <id>14664094</id>
    </contributor>
    <comment>Redirected to main topic</comment>
    <text xml:space="preserve">#REDIRECT [[Article A]]</text>
    <sha1>k77e50uxik162v2pjys615dq2kjyehb</sha1>
    <model>wikitext</model>
    <format>text/x-wiki</format>
</revision>
</page>


<page>
<title>Article b</title>
<ns>0</ns>
<id>11858</id>
<redirect title="Article B"/>
<revision>
    <id>518833992</id>
    <parentid>511689418</parentid>
    <timestamp>2012-10-20T06:12:35Z</timestamp>
    <contributor>
        <username>M0rphzone</username>
        <id>14664094</id>
    </contributor>
    <comment>Redirected to main topic</comment>
    <text xml:space="preserve">#REDIRECT [[Article B]]</text>
    <sha1>k77e50uxik162v2pjys615dq2kjyehb</sha1>
    <model>wikitext</model>
    <format>text/x-wiki</format>
</revision>
</page>