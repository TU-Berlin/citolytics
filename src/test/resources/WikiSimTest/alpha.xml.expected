-5873957629957137106|Article A|Article B|3|1|0.5773502691896257|0.3720410580113015|0.1111111111111111|3.0
-5873138001670156304|Article B|Article C|83|2|0.31993474978374437|0.0749013328693349|0.001467109687116807|83.0
-5873138001670156303|Article B|Article D|89|1|0.105999788000636|0.017601472199121834|1.2624668602449185E-4|89.0
-5872318373383175502|Article C|Article D|59|1|0.13018891098082386|0.025481981115495565|2.872737719046251E-4|59.0
6238752168419089436|Article E|SeeAlso Article 2|49|1|0.14285714285714285|0.030117819624378612|4.1649312786339027E-4|49.0
-5873957629957137104|Article A|Article D|5|1|0.4472135954999579|0.2349237886176038|0.04|5.0
-5873138001670156302|Article B|Article E|92|3|1.13367280980758|0.6255065078852046|0.15125283446712018|92.0
-5873138001670156274|Article B|Article a|11|1|0.30151134457776363|0.11554378320092187|0.008264462809917356|11.0
5814050287007184600|Article B|SeeAlso Article 1|2|1|0.7071067811865475|0.5358867312681466|0.25|2.0
9151080181846839682|SeeAlso Article 1|SeeAlso Article 2|33|1|0.17407765595569785|0.04298703134869941|9.182736455463728E-4|33.0
-6625346389269818089|Article A|SeeAlso Article 1|52|1|0.1386750490563073|0.028549400346575122|3.698224852071006E-4|52.0
-6625346389269818088|Article A|SeeAlso Article 2|63|1|0.12598815766974242|0.024021133976808145|2.519526329050139E-4|63.0
-6200644507857913254|Article D|SeeAlso Article 1|144|2|0.23966448919028388|0.044250506838602814|4.399048605095357E-4|144.0
-6200644507857913253|Article D|SeeAlso Article 2|54|1|0.13608276348795434|0.027595966950786033|3.4293552812071334E-4|54.0
-5873957629957137074|Article A|Article b|28|1|0.1889822365046136|0.04983767478624899|0.0012755102040816328|28.0
-5871498745096194700|Article D|Article E|5|1|0.4472135954999579|0.2349237886176038|0.04|5.0
-193297110425364327|Article C|SeeAlso Article 1|46|2|0.42468449690012944|0.12401219492737747|0.004361929957168052|46.0
-193297110425364326|Article C|SeeAlso Article 2|18|2|0.7245636936125726|0.3343385748315139|0.04591715976331361|18.0
5814050287007184601|Article B|SeeAlso Article 2|101|2|0.29212234192503606|0.06380599652945161|0.0010458949419988382|101.0
6238752168419089435|Article E|SeeAlso Article 1|117|2|0.2794623770205499|0.05971799817250503|9.65047530017361E-4|117.0
-5873957629957137105|Article A|Article C|104|2|0.27750411972526384|0.057171119683352305|7.429355281207133E-4|104.0
-5872318373383175501|Article C|Article E|107|2|0.27344332743664335|0.055660105603999394|6.989341041264093E-4|107.0
-5872318373383175472|Article C|Article b|82|1|0.11043152607484655|0.018948181359429297|1.48720999405116E-4|82.0
-5870679116809213871|Article E|Article a|16|1|0.25|0.08246924442330589|0.00390625|16.0